attributes/default.rb

```diff
@@ -22,13 +22,12 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-# AWS Placeholders
-default['incident_bot']['aws'].tap do |aws|
-  aws['redis_bucket'] = ''
-  aws['secret_key'] = ''
-  aws['access_key'] = ''
-  aws['domain'] = ''
-end
+# AWS
+default['incident_bot']['aws']['domain'] = ''
```

Comment:
This is accessed on line 80 of this file.  However I cannot comment on that line since that section did not change.  It is normally customary to try to not use attributes to set other attributes as described here: https://coderanger.net/derived-attributes/  I am not a fan of lazy attributes either as described in the article.  The portion below the image here is a good read: https://docs.chef.io/chef_client.html#the-chef-client-title-run.  Another option is to set this using `node.default...` in the recipe.  The reason i bring this up is that I think since this is set to empty by default and is essential it should be checked for empty in the logic somehow. And putting if/then logic inside of an attribute file is icky.